lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -303,7 +303,10 @@ export abstract class ClientApplication {
 
             const redirectStartPage = (request && request.redirectStartPage) || window.location.href;
             // Show the UI once the url has been created. Response will come back in the hash, which will be handled in the handleRedirectCallback function.
-            return interactionHandler.initiateAuthRequest(navigateUrl, authCodeRequest, this.config.system.redirectNavigationTimeout, redirectStartPage);
+            return interactionHandler.initiateAuthRequest(navigateUrl, authCodeRequest, {
```

Comment:
nit: Can we define the params above and make this a one liner?

Reply:
done.